package.json

```diff
@@ -0,0 +1,20 @@
+{
+	"scripts": {
+		"lint": "eslint *.js --quiet"
```

Comment:
Is the whole intention of adding package.json and yarn.lock to add linting to this project?  How is the typical engineer supposed to use this to download depencies and run this script?  No documentation has been added that helps a new engineer to the project understand how to use this - why was that left out?